lib/services/serviceBus/package.json

```diff
@@ -11,7 +11,7 @@
     "Tavares, Chris <ctavares@microsoft.com>",
     "Kulshrestha, Ankur <ankurkul@microsoft.com>"
   ],
-  "version": "0.10.5",
+  "version": "0.10.6",
```

Comment:
Why dont you update the minimum version of azure-common dependency to "^0.9.19" over here as well.

Reply:
Made the change in other pull request

Reply:
@amolr - Can you update `"azure-common": "^0.9.19"` dependency in this PR. I have closed the other PR.